src/featurestorebundle/delta/feature/NullHandler.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from pyspark.sql import DataFrame
+from pyspark.sql import functions as f
+from featurestorebundle.feature.FeatureList import FeatureList
+
+
+class NullHandler:
+    def fill_nulls(self, df: DataFrame, feature_list: FeatureList) -> DataFrame:
+        self.__check_default_values_valid(feature_list)
+
+        fill_dict = {
+            feature.name: feature.template.default_value for feature in feature_list.get_all() if feature.template.default_value is not None
+        }
+
+        return df.fillna(fill_dict)
+
+    def to_storage_format(self, df: DataFrame, feature_list: FeatureList) -> DataFrame:
+        for feature in feature_list.get_all():
+            if feature.template.default_value is None:
+                df = df.withColumn(feature.name, f.create_map(f.lit(0), f.col(feature.name)).cast(feature.storage_dtype))
```

Comment:
isn't it maybe better to make a list of Columns and then select them at once? this is slower according to Olda :)

Reply:
True